### PR TITLE
php.ini config include

### PR DESCRIPTION
**Pull request by allensea**
Can now add config to the end of php.ini directly by setting the pillar item php_fpm.additional_config

### CODE REVIEW

File: php-fpm/templates/php.ini [hunk 1]
```diff
@@ -1700,3 +1700,9 @@ soap.wsdl_cache_limit = 5
 ; Local Variables:
 ; tab-width: 4
 ; End:
+
+[additional_config]
+; Include additional config from salt pillar if specified
+{% if php_fpm.additional_config %}
+{{ php_fpm.additional_config }}
+{% endif %}
```
Comment (mikepea) [inline php-fpm/templates/php.ini]: :cop: no newline at end of the file

File: php-fpm/pecl/xdebug.sls [hunk 2]
```diff
@@ -4,4 +4,5 @@ include:
   - repos
 
 xdebug:
-  pecl.installed
+  pecl.installed:
+    - defaults: 'True'
```
Comment (mikepea) [inline php-fpm/pecl/xdebug.sls]: This should be a separate commit. Why is it in here?

Comment (allensea) [inline php-fpm/pecl/xdebug.sls]: Because I'm lazy
